reporters_db/data/reporters.json

```diff
@@ -103,8 +103,7 @@
                 "A.F.T.R.": "A.F.T.R. (RIA)",
                 "A.F.T.R. 2d (RIA)": "A.F.T.R.2d (RIA)",
                 "A.F.T.R. 2d": "A.F.T.R.2d (RIA)",
-                "A.F.T.R.2d": "A.F.T.R.2d (RIA)",
-                "P-H T.C. Memo.": "P-H T.C. Memo."
+                "A.F.T.R.2d": "A.F.T.R.2d (RIA)"
```

Comment:
Shouldn't this be: `"P-H T.C. Memo.": "A.F.T.R. (RIA)",`. Confused by it disappearing.

Reply:
You are correct, this has been fixed.